src/app/games/rectanglz/RectanglzState.ts

```diff
@@ -0,0 +1,64 @@
+import { Coord } from 'src/app/jscaip/Coord';
+import { GameStateWithTable } from 'src/app/jscaip/GameStateWithTable';
+import { Player, PlayerOrNone } from 'src/app/jscaip/Player';
+import { PlayerNumberMap } from 'src/app/jscaip/PlayerMap';
+import { Table } from 'src/app/utils/ArrayUtils';
+
+export class RectanglzState extends GameStateWithTable<PlayerOrNone> {
+
+    public static mapper: (oldState: RectanglzState, newBoard: Table<PlayerOrNone>) => RectanglzState =
```

Comment:
renommer mapper en withBoard

Reply:
puis il est pas utilisé ?

Reply:
"F-U I WON'T DO WHAT YOU TELL ME (x3)"
Coding in the name of !